helm/values.yaml

```diff
@@ -0,0 +1,110 @@
+# Default values for substrate-telemtry.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+replicaCount:
+  shard: 3
+  core: 1
+
+image:
+  repository: parity/substrate-telemetry-backend
+  pullPolicy: IfNotPresent
```

Comment:
Shouldn't this be "always" if the image is meant to update on new releases? I may be misunderstanding, though.

Reply:
The image tag is specified explicitly as arguments for the helm command and it overrides this one in values.yaml file in a way that for each commit, we would have a unique docker image that would be used for that deployment. Setting `pullPolicy: IfNotPresent` would prevent images from being downloaded when deploying a previous commit on the cluster.

Reply:
i think one should not rely on the fact that the image has been pulled before. the node pool could have been updated and then alertering the deployment could lead to issues that are harder to debug. but if you set the deployment tag explicitly it should be fine.

Reply:
I've had annoying-to-debug issues in the past when not using `Always`; it feels safer to err on the side of caution?